net/pfSense-pkg-cellular/files/usr/local/pkg/cellular.inc

```diff
@@ -60,7 +64,34 @@ function cellular_sync() {
 
 	foreach (["port", "baudrate", "timeout", "initstring"] as $key) {
 		$key_val = ($cellular[$key] != "default" && $cellular[$key] != "") ? $cellular[$key] : "";
-		exec(PYTHON_BIN . ' ' . INTERFACE_BIN . ' $device $baudrate $timeout setcfg Interface $key $key_val signal');
+		exec(PYTHON_BIN . ' ' . INTERFACE_BIN . ' setcfg Interface ' . $key . ' ' . $key_val);
+	}
+
+	#Cron
+	install_cron_job('cellular_check.php', false);
```

Comment:
This will trigger write_config() on every package resync. How about using else when the package not enabled?

Reply:
The reason why I did it this way is that the cron job script has different command line options like -r -i and so on.
If the user changes them in the GUI and the cron job options change I have no way of detecting the cron job that is already in place and replace it.
I will try and find a better way to detect the old cron job.

Reply:
Ok, I pushed a check for this, please let me know if that is better.

Best
Sven